app/products/playbooks/actions/local/checklist.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import DatabaseManager from '@database/manager';
+import {getPlaybookChecklistItemById} from '@playbooks/database/queries/item';
+import {logError} from '@utils/log';
+
+export async function updateChecklistItem(serverUrl: string, itemId: string, state: ChecklistItemState) {
+    try {
+        const {database} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
+        const item = await getPlaybookChecklistItemById(database, itemId);
+        if (!item) {
+            return {error: 'Item not found'};
+        }
+
+        await database.write(async () => {
+            item.update((i) => {
+                i.state = state;
```

Comment:
What about the sync status? I know we are not using it right now, but perhaps we should tackle that here?

Reply:
I think it is better to "do it right once", once we know exactly how we want that to look, than guess now and refactor later. Right now we are not doing any kind of "optimistic approach", so at this point it should be "synced = true", but 0/5.